techeria_controller/templates/contact.html

```diff
@@ -13,7 +13,7 @@
   <meta name="description" content="" />

```

Comment:
This contact page shouldn't have an image of a watch. I will add the change to make sure the contact page only has an image related to technology